runtime/doc/options.txt

```diff
@@ -2438,7 +2438,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 'foldcolumn' 'fdc'	string (default "0")
 			local to window
 	When and how to draw the foldcolumn. Valid values are:
-	    "auto":       resize to the maximum amount of folds to display.
```

Comment:
`auto` should work so you found a bug. I'm going to make this a feature.  cc @teto

Reply:
@janlazo See https://github.com/neovim/neovim/issues/12278#issuecomment-626894803

Reply:
I know. I'm going to update the code to alias it to `auto:9`. Useful to hide the internal limit because max foldcolumns was reduced from 12 to 9.

Reply:
@janlazo Shouldn't it be aliased to `auto:1` instead?
Because `signcolum=auto` means `signcolumn=auto:1`, right?

Reply:
The goal was to have `signcolumn` behave like `signcolumn`. 
Seems like signcolumn=auto behaves like signcolumn=auto:1 (to confirm) so we should do the same for foldcolumn

Reply:
https://github.com/neovim/neovim/blob/master/src/nvim/option.c#L7097-L7128
`set signcolumn=auto` behaves as `set signcolumn=auto:1`.